source/test/runbook_validator.test.ts

```diff
@@ -135,7 +132,7 @@ function getRunbooksFromDirectories(directories: string[], exclusions: string[])
 }
 
 function getControlRunbooks(runbooks: RunbookTestHelper[]): RunbookTestHelper[] {
```

Comment:
Isn't this function unnecessary?
We can do `runbooks.filter(it => it.isRemediationRunbook())` in line 181 and remove this function.